buildscripts/test-logging.properties

```diff
@@ -0,0 +1,7 @@
+handlers= java.util.logging.ConsoleHandler
+.level= INFO
```

Comment:
formatting here is slightly different than the rest of the file. missing a space before the `=` sign.